azure/template.json

```diff
@@ -275,13 +345,15 @@
             },
             "dependsOn": [
                 "worker-app-service-plan",
-                "storage-account"
+                "storage-account",
+                "function-app-subnet"
```

Comment:
The dependency doesn't need to be declared, it's created implicitly by using the reference function